lua/toggleterm/terminal.lua

```diff
@@ -298,7 +302,12 @@ end
 --- @return number
 function M.identify(name)
   name = name or api.nvim_buf_get_name(api.nvim_get_current_buf())
-  local parts = vim.split(name, "#")
+  local parts
+  if vim.fn.has('win32') then
+    parts = vim.split(name, "::")
```

Comment:
could this be determined once on startup since it's static otherwise e.g.
```lua
-- At the top of the file
local cmd_sep = has('win32') and "::" or "#"

--- then in the func
local parts = vim.split(name, cmd_sep)
```